src/modelio.cs

```diff
@@ -113,6 +113,15 @@ interface MDLAsset : NSCopying
 		[Wrap ("GetComponent (new Protocol (type))")]
 		IMDLComponent GetComponent (Type type);
 
+		[iOS (10,3), TV (10,2), Mac (10,12,4)]
+		[Export ("objectForKeyedSubscript:")]
+		[return: NullAllowed]
+		IMDLComponent GetObject (Protocol key);
+
+		[iOS (10,3), TV (10,2), Mac (10,12,4)]
+		[Export ("setObject:forKeyedSubscript:")]
+		void SetObject ([NullAllowed] IMDLComponent obj, Protocol key);
+
```

Comment:
Check with @VincentDondain (or his PR) IIRC there was a reason for those not being there